napari/utils/progress.py

```diff
@@ -75,7 +73,7 @@ def __init__(
 
         # get progress bar added to viewer
         try:
-            from .dialogs.activity_dialog import get_pbar
+            from .._qt.dialogs.activity_dialog import get_pbar
 
             pbar = get_pbar(self, nest_under=nest_under, **pbar_kwargs)
         except ImportError:
```

Comment:
It'd be better to have the try..import at the module level and not in a function.  One approach may be to treat `utils.progress()` as a factory function returning a tdqm-like object.  This might look something like:

```python
# At the module level in utils.py
try:
   import .._qt.dialogs.activity_dialog import qet_pbar
   def progress(*args,**kwargs):
       pbar=get_pbar(*args,**kwargs)
       return pbar if pbar else tdqm(*args,**kwargs)
except ImportError:
   def progress(*args,**kwargs):
       return tdqm(*args,**kwargs)
```

I think this would also simplify some of the conditionals in the progress class below.